.release-please-config.json

```
@@ -0,0 +1,26 @@
+{
+  "release-type": "python",
+  "bump-minor-pre-major": true,
+  "bump-patch-for-minor-pre-major": true,
+  "separate-pull-requests": true,
+  "always-update": true,
+  "changelog-type": "github",
+  "include-component-in-tag": true,
+  "include-v-in-tag": false,
+  "draft": true,
```

This should be `draft-pull-request` instead. 
```suggestion
  "draft-pull-request": true,
```